app/src/main/java/org/wikipedia/views/ImagePreviewDialog.kt

```diff
@@ -104,39 +103,56 @@ class ImagePreviewDialog : ExtendedBottomSheetDialogFragment(), DialogInterface.
             // Show the image description when a structured caption does not exist.
             addDetailPortion(getString(R.string.suggested_edits_image_preview_dialog_description_in_language_title,
                     WikipediaApp.getInstance().language().getAppLanguageLocalizedName(suggestedEditsSummary.lang)),
-                    suggestedEditsSummary.description, false)
+                    SpannableString(suggestedEditsSummary.description), null)
```

Comment:
I think we can just use `view.detailTextView.text = StringUtil.strip(StringUtil.fromHtml(detail))`. Please see the following comment.